### DISCUSSION TITLE
can u make an installation video please?

### DISCUSSION THREAD

**Discussion by iLayzZ** [gucci-on-fleek/lockdown-browser#24] [Q&A] [answered]
i have all my exams on lockdownbrowser and i also use blackboard. i used to have a self coded software that i bought of someone but it doesnt work anymore. now i was searching on github and founded yours. can u make an installation video? it would be really helpfull! thankyou

**Comment by gucci-on-fleek** [comment, marked as answer]
There's already a [demo video](https://github.com/gucci-on-fleek/lockdown-browser/tree/master#lockdown-browser-in-windows-sandbox) near the top of the `README`; right below the screenshot, you can click the “View Demo Video” text to see it. The video doesn't show the entire installation process, but I don't think that a video would be helpful with the remaining/omitted steps.

Here's some commentary on the installation steps from the main instruction page.

1. ### [Install the prerequisites](https://github.com/gucci-on-fleek/lockdown-browser#system-requirements)

   This mostly consists of clicking two links, downloading the installers, and then running the installers. You can't easily link to the installation pages from a video, and downloading/running the installers is pretty much just clicking “Next” a bunch of times to just select the default installation options. 

2. ### [Build the program](https://github.com/gucci-on-fleek/lockdown-browser#building)

   The entirety of the text of this section:

   > Clone the repository, then run build.ps1. Then, [install the _Windows Sandbox_](https://www.howtogeek.com/399290/how-to-use-windows-10s-new-sandbox-to-safely-test-apps/). That's it!

   The link there contains _very_ detailed instructions on how to install the _Sandbox_, so I doubt that I could do any better than that. And the demo video already shows cloning and building.

3. ### [Run the program](https://github.com/gucci-on-fleek/lockdown-browser/tree/master#running)

   The steps here are to build the project (which is exactly points 1 and 2 above), download the _Browser_ (which I can't help with since every school has a different way to do this), and then double-click a file (which I doubt you need a video to know how to do). But regardless, this is all shown in the same demo video linked to above.

4. That's it; there are no more steps.
